packages/components/shared/src/popover.ts

```diff
@@ -51,24 +51,30 @@ export const positionPopover = (
       flip(),
       options.viewportMargin !== undefined ? shift({ padding: options.viewportMargin }) : undefined,
       size({
-        // With popover, we no longer need to worry about any clipping ancestors
-        boundary: document.documentElement,
+        // With popover, we no longer need to
         padding: options.viewportMargin,
-        apply: ({ availableWidth, availableHeight }) => {
+        apply: ({ availableWidth, availableHeight, elements }) => {
           // Make sure that the overlay is contained by the visible page.
-          const maxBlockSize =
-              Math.max(MIN_OVERLAY_HEIGHT, Math.floor(availableHeight)) - (options.viewportMargin ?? 0),
-            maxInlineSize = options.maxWidth ?? Math.floor(availableWidth);
-
           const style = getComputedStyle(element),
-            currentMaxBlockSize = parseInt(style.maxBlockSize) ?? 0,
+            max = style.getPropertyValue('--sl-popover-max-block-size'),
+            currentMaxBlockSize = !isNaN(parseInt(max)) ? parseInt(max) : 0,
+            min = style.getPropertyValue('--sl-popover-min-block-size'),
```

Comment:
It looks like the tooltip is currently too high

<img width="715" alt="image" src="https://github.com/user-attachments/assets/eb9f71f5-6285-48e6-8b17-c409a5aa8bc0">